tests/pyupgrade_test.py

```diff
@@ -1448,6 +1448,39 @@ def test_fix_native_literals(s, expected):
     assert _fix_py3_plus(s) == expected
 
 
+@pytest.mark.parametrize(
+    ('s', 'expected'),
+    (
+        ('"asd".encode("utf-8")', '"asd".encode()'),
+        ('"asd".encode("utf8")', '"asd".encode()'),
+        ('"asd".encode("UTF-8")', '"asd".encode()'),
+        (
+            'sys.stdout.buffer.write(\n    "a"\n    "b".encode("utf-8")\n)',
+            'sys.stdout.buffer.write(\n    "a"\n    "b".encode()\n)',
+        ),
+    ),
+)
+def test_fix_encode(s, expected):
+    assert _fix_py3_plus(s) == expected
+
+
+@pytest.mark.parametrize(
+    's',
+    (
+        # see https://git.io/fjRcs
```

Comment:
let's just write out the comment, the link doesn't seem to work anyway
```python
# for now, pyupgrade is timid in this particular case
```

Reply:
could also just move this down below the other comment about timidity (is that a word?)

Reply:
sure